test/core/tsi/s2a/s2a_test_util.h

```diff
@@ -38,4 +38,12 @@ grpc_byte_buffer* create_example_session_state(bool admissible_tls_version,
 
 size_t expected_message_size(size_t plaintext_size);
 
+bool check_encrypt_record(TLSCiphersuite ciphersuite, uint8_t* record_one,
```

Comment:
Can you please add a comment explaining what this function and the one below do? Thanks!

Reply:
Done.